fastkml/times.py

```diff
@@ -196,6 +199,7 @@ def __init__(
             **kwargs,
         )
         self.timestamp = timestamp
+        self.when = when
```

Comment:
`when` and `timestamp` should be mutually exclusive

Reply:
Hi @cleder,
Would you suggest I instead accept a parameter called `pointInTime` _(I'm open to an alternate name)_, which is of type `KmlDateTime | When` so that users can only provide either a value of type `When` or `KmlDateTime` but not both?